frontend/lib/routes.ts

```diff
@@ -56,6 +60,33 @@ function createIssuesRouteInfo(prefix: string): IssuesRouteInfo {
   };
 }
 
+export type OnboardingRouteInfo = ReturnType<typeof createOnboardingRouteInfo>;
```

Comment:
Just found out about `ReturnType`--it makes creating ad-hoc typings like this so much easier.